superset/assets/src/datasource/DatasourceModal.jsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Alert, Button, Modal } from 'react-bootstrap';
+import Dialog from 'react-bootstrap-dialog';
+
+import { t } from '../locales';
+import DatasourceEditor from '../datasource/DatasourceEditor';
+import withToasts from '../messageToasts/enhancers/withToasts';
+
+const $ = window.$ = require('jquery');
```

Comment:
do you have to set `window.$`? this is a pattern that appears in many files and I don't think is necessary.

Reply:
yes I'm not sure what the intent was originally... copy pasta